clients/tabby-agent/src/codeCompletion/contexts.ts

```diff
@@ -203,6 +205,16 @@ export class CompletionContext {
       })
       .sort((a, b) => b.score - a.score);
 
+    const snippetsOpenedFiles = this.snippetsFromOpenedFiles
+      ?.map((snippet) => {
+        return {
+          filepath: snippet.filepath,
+          body: snippet.text,
+          score: snippet.score,
+        };
+      })
+      .sort((a, b) => b.score - a.score);
```

Comment:
The comments in https://github.com/TabbyML/tabby/blob/main/crates/tabby/src/services/completion.rs#L147 says that the snippets should have been deduplicated at client-side.  
It seems that this is not implemented. Please leave a FIXME if it is not planned in this PR.